### ISSUE TITLE
[Question] Is There a Way to Send Break Signal

### ISSUE THREAD

**Issue by KiddoV** [bugst/go-serial#131]
Thank you for the great package. I would like to send a break signal to the port. How do I accomplish this?
It like when using TeraTerm you send the port a break signal.
Thank you!

**Comment by kroppt** [comment]
According to [the manual](https://ttssh2.osdn.jp/manual/4/en/macro/command/sendbreak.html), break signals are just two bytes, a 255 byte followed by a 243 byte.
> This macro sends IAC BREAK(255 243) to the host system on the TELNET console.

So, you should be able to replicate a break signal by sending those bytes.

**Comment by KiddoV** [comment]
@kroppt thanks for the info. Can you give some code example on how to send those bytes? I appreciate it!

**Comment by kroppt** [comment]
See https://pkg.go.dev/go.bug.st/serial#section-documentation for all the setup.

Then, when you are writing to the serial port, send the two bytes:
```go
n, err := port.Write([]byte{255, 243})
```

**Comment by KiddoV** [comment]
Thanks! I will try that.

**Comment by KiddoV** [comment]
@kroppt Is there anyway that I can get the ``syscall.Handle`` without modify the lib? I want to try to set comm break like this:
https://github.com/ricorx7/go-serial/blob/be02e536a880/syscall_windows.go#L65
```go
func SetCommBreak(handle syscall.Handle) (err error) {
	r1, _, e1 := syscall.Syscall(procSetCommBreak.Addr(), 1, uintptr(handle), 0, 0)
	if r1 == 0 {
		if e1 != 0 {
			err = error(e1)
		} else {
			err = syscall.EINVAL
		}
	}
	return
}
```

**Comment by kroppt** [comment]
No, the type `windowsPort` is unexported.

**Comment by KiddoV** [comment]
Do you have any suggestion of how I can use the ``SetCommBreak`` from ``syscall.NewLazyDLL`` with my ``serial.Port`` instance?

**Comment by kroppt** [comment]
No. That would be the same as the code snippet you posted.

**Comment by schmidtw** [comment]
I don't have a way to put a scope on the output, but the suggested solution of sending 255,243 doesn't appear to work in all cases.  I have an FTDI serial chip on a linux system.  When I send a break using the snippet below, I'm able to see the expected results, but when I send 255,243 I don't.

```golang
unix.Syscall(unix.SYS_IOCTL, mySerialPortFile.Fd(), uintptr(unix.TCSBRKP), uintptr(0))
```
In my use case I need to reset a DS2480 chip & one way to do this is to send a break for 2ms+.

**Comment by supaplextor** [comment]
Looks like in this  pull request: #137

**Comment by KiddoV** [comment]
Sadly it doesn't support Windows 🥲

**Comment by cmaglie** [comment]
@KiddoV please give #145 a try.

**Comment by cmaglie** [comment]
Fixed by #145
